src/components/Item.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import db from '../lib/firebase';
+
+const Item = ({ name, id, lastPurchasedDate }) => {
+  const checkHandler = () => {
```

Comment:
nit: I don't know what the typical pattern is or if this is just me, but I've always seen these functions match their html event names. So `onChange={onChange}` which makes it really clear what this function is responsible for.

Reply:
I think its a choice thing. For me I tend to preface my events functions with `handle` eg `handleChange` or `handleChecked` in this case. I think any descriptive name is fine.

Reply:
I see, that works. My experience is using `onChange` or even `handleOnChange` where it can call a whole series of functions if necessary, but we know exactly where it starts. If I were reading the function list of a file or component, it's simpler to jump to the `handleOnChange` function when I need to handle the onChange event rather than going to the component to find out what function it's calling. Totally a preference thing though. 👍